doc/classes/CodeEdit.xml

```diff
@@ -458,23 +458,23 @@
 			<param index="0" name="line" type="int" />
 			<param index="1" name="bookmarked" type="bool" />
 			<description>
-				Sets the line as bookmarked.
+				Sets the line as bookmarked. If [code]true[/code] and [member gutters_draw_bookmarks] is [code]true[/code], shows the [theme_item bookmark] icon in the gutter for this line. See [method get_bookmarked_lines] and [method is_line_bookmarked].
 			</description>
 		</method>
 		<method name="set_line_as_breakpoint">
 			<return type="void" />
 			<param index="0" name="line" type="int" />
 			<param index="1" name="breakpointed" type="bool" />
 			<description>
-				Sets the line as breakpointed.
+				Sets the line as breakpointed. If [code]true[/code] and [member gutters_draw_breakpoints_gutter] is [code]true[/code], shows the [theme_item breakpoint] icon in the gutter for this line. See [method get_breakpointed_lines] and [method is_line_breakpointed].
```

Comment:
```suggestion
				Sets the line as a breakpoint. If [code]true[/code] and [member gutters_draw_breakpoints_gutter] is [code]true[/code], displays the [theme_item breakpoint] icon in the gutter for this line. See [method get_breakpointed_lines] and [method is_line_breakpointed].
```

Reply:
I think I will use 'is drawn' instead of 'displays' or 'shows', to better match other mentions of icon.

Reply:
Yeah, good choice to keep the wording consistent